### PR TITLE
VZ-4810 Fluentd parsing for verrazzano operators

### PR DESCRIPTION
**Pull request by pfmackin**
Add parser for VPO, VO, and VMO.
Fix level parsing for Klog

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 1]
```diff
@@ -394,10 +394,10 @@ data:
           format json
           time_key @timestamp
           time_format %Y-%m-%dT%H:%M:%S.%NZ
-       </pattern>
+        </pattern>
```
Comment (davisgomes) [inline platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:397]: Extra space?

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 2]
```diff
@@ -394,10 +394,10 @@ data:
...
         # Kubernetes klog format
         <pattern>
-          format /^(?<level>\w)(\d{2}\d{2}) (?<logtime>\d{2}:\d{2}:\d{2}.\d{6})\s*?(?<message>[\s\S]*?)$/
+          format /^(?<level>.)(\d{2}\d{2}) (?<logtime>\d{2}:\d{2}:\d{2}.\d{6})\s*?(?<message>[\s\S]*?)$/
```
Comment (kminder) [inline platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:400]: What use case required you to change this from /w to .
Also is the message really the message or is the thread id, and caller getting included in the message?

Comment (davisgomes) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:400]: I think the log level for klog is always one character, so this should always match the log level character.

Comment (kminder) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:400]: Yes but \w is usually defined as [A-Za-z0-9_].  Why would you need more?

**Review by kminder — APPROVED** [review]
Approved as there will be a follow up PR to address these issues and add a test.

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml (hunk 2 above)
Comment (pfmackin) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:400]: The previous pattern didn't work, i tested it and discussed with Bob.  I used a pattern that he said worked and, in fact, it did work once i changed it.

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml (hunk 1 above)
Comment (pfmackin) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:397]: it wasn't out of alignment before.

**Pull request merged**